.github/workflows/check.yaml

```diff
@@ -34,3 +35,41 @@ jobs:
     uses: insightsengineering/r.pkg.template/.github/workflows/build-check-install.yaml@main
     if: >
       needs.job-setup.outputs.check_out == 'true'
+
+  r-cmd-with-docker:
```

Comment:
@pawelru / @walkowif this is the new exercice using docker container instead of runner